src/wallet/wallet.cpp

```diff
@@ -2083,7 +2083,7 @@ CAmount CWalletTx::GetAvailableCredit(bool fUseCache) const
             const CTxOut &txout = tx->vout[i];
             nCredit += pwallet->GetCredit(txout, ISMINE_SPENDABLE);
             if (!MoneyRange(nCredit))
-                throw std::runtime_error(std::string(__func__) + ": value out of range");
+                throw std::runtime_error(std::string(__func__) + " : value out of range");
```

Comment:
this looks like a mistake, I would revert it

Reply:
dropped